Description: Display the tests on the console as well in the log file.
Author: Chuck Short <zulcss@ubuntu.com>
Forwarded: Yes
diff -Naurp nova-2013.1.orig/run_tests.sh nova-2013.1/run_tests.sh
--- nova-2013.1.orig/run_tests.sh	2012-11-07 12:06:53.000000000 -0600
+++ nova-2013.1/run_tests.sh	2012-11-08 08:06:11.797024712 -0600
@@ -88,7 +88,7 @@ function run_tests {
   # Cleanup *pyc
   ${wrapper} find . -type f -name "*.pyc" -delete
   # Just run the test suites in current environment
-  ${wrapper} $NOSETESTS | tee nosetests.log
+  ${wrapper} $NOSETESTS 2>&1 | tee nosetests.log
   # If we get some short import error right away, print the error log directly
   RESULT=$?
   if [ "$RESULT" -ne "0" ];
